plugins/coverimage.koplugin/main.lua

```diff
@@ -17,6 +17,7 @@ local UIManager = require("ui/uimanager")
 local RenderImage = require("ui/renderimage")
 local Screen = require("device").screen
 local T = require("ffi/util").template
+local Util = require("util")
```

Comment:
```suggestion
```

Reply:
I see, that `util` is already included. Thanks.